### PR TITLE
Updated versions, dates, rotated authorship

### LINKED ISSUE #557: Create the METviewer v6.0.0 release
*Replace italics below with details for this issue.*

## Describe the Task ##
Follow the Release Guide in METplus Read the Docs to generate the coordinated v3.0.0 release

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Review default **alert** labels
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Milestone and Projects ###
- [ ] Select **Milestone** as a **METviewer-X.Y.Z** version, **Consider for Next Release**, or **Backlog of Development Ideas**
- [ ] For a **METviewer-X.Y.Z** version, select the **METviewer-X.Y.Z Development** project

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the next official version
Select: **METviewer-X.Y.Z Development** project for development toward the next official release
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by bikegeek**
## Pull Request Testing ##

- [ ] Describe testing already performed for these changes:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [ ] Do these changes introduce new SonarQube findings? **[Yes or No]**</br>
If **yes**, please describe:

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **METviewer-X.Y.Z Development** project for official releases
- [ ] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Comment by jprestop** [comment]
@bikegeek For some reason, the 'develop' release notes page for METviewer has not only the betas and rc1 for 6.0.0, but also still has the release notes for 5.1.0, 5.0.1, and 5.0.0. Unfortunately, when you were consolidating the notes, issues from those old releases were included too.  I'll go in and clean it up, removing the issues from those older releases since you're cutting all four Analysis repos this time.  I'll continue to review and approve soon, hopefully.

**Comment by bikegeek** [comment]
sorry about that, I can clean that up right now...

**Comment by jprestop** [comment]
@bikegeek No, it's done!  I just did it.  It's all good.

File: docs/conf.py [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
 author = 'UCAR/NCAR'
-author_list = "Burek, T., T. Jensen, I. Srivastava, M. Win-Gildenmeister, L. Goodrich, J. Prestopnik, L. Blank, and T. Fowler"
-version = "3.0.0-rc1"
+author_list = "Jensen, T., I. Srivastava, M. Win-Gildenmeister, L. Goodrich, J. Prestopnik, L. Blank, T. Fowler, and T. Burek"
+version = "3.0.0"
 verinfo = version
 release = f'{version}'
 release_year = '2024'
-release_date = f'{release_year}-11-13'
+release_date = f'{release_year}-12-18'
```
Comment (jprestop) [inline docs/conf.py:27]:
```suggestion
release_date = f'{release_year}-12-19'
```
Modified to match the release notes documentation date. @bikegeek if you do plan to release tonight, please modify the date in the release-notes.rst file to be the 18th and don't change this reference.  :)

**Comment by bikegeek** [comment]
Thanks, that is strange, I wonder why those didn't show up in my editor?
Thanks for getting that done so quickly.

**Review by jprestop — APPROVED** [review]
Thanks for all of your work on this task, @bikegeek.  I appreciate it.  I approve this request.

**Pull request merged**